bin/postgres-ha/bootstrap/pre-bootstrap.sh

```diff
@@ -318,8 +318,46 @@ build_bootstrap_config_file() {
     # postgresql.conf file
     if [[ "${PGHA_TLS_ENABLED}" == "true" ]]
     then
+      echo_info "Enabling TLS"
+      pghba_tls_file="/opt/cpm/conf/postgres-ha-pghba-tls.yaml"
+
+      # if there is a TLS keypair for a replication user detected, we will need
+      # to copy this to a special mount in order to properly enable
+      # certificate-based authentication between replicas. This is due to
+      # PostgreSQL requiring a client's TLS key to have permissions no less strict
+      # than 0600. In this case, the client is the "postgres" user, and because
+      # Kubernetes Secrets are owned by the group, we need to create a copy of
+      # this key with the correct permissions
+      if [[ -f "/pgconf/tls/tls-replication.key" ]] && [[ -f "/pgconf/tls/tls-replication.crt" ]]
+      then
+        echo_info "Enabling certificate-based authentication for replication"
+        touch /pgconf/tls-replication/tls.key /pgconf/tls-replication/tls.crt
```

Comment:
@andrewlecuyer `/pgconf/tls-replication` is an `emptyDir`, see: [https://github.com/CrunchyData/postgres-operator/pull/1662/files#diff-f3ecf8804e14f07041039de1d908c047](https://github.com/CrunchyData/postgres-operator/pull/1662/files#diff-f3ecf8804e14f07041039de1d908c047)

Reply:
Thanks for the clarification.  I assume the fact that other ConfigMaps and Secrets are using the `pgconf` directory doesn't impact the the writeability of the `emptyDir` under `pgconf`?

Also, would it makes sense to just use another existing directory on the file system, e.g. `tmp` or `crunchyadm`, instead of mounting another `emptyDir`?

Reply:
@andrewlecuyer AFAICT it does not impact the writability, given I've tested this and it works in my Kubernetes environment. I will admit to it needing testing in OpenShift.

It feels grotty to me to put TLS credentials into `/tmp` and it would make me wonder what happens if someone wants to run a readOnly root FS (see [ch7571] -- and technically we can't support this fully yet).

Reply:
Doesn't need to be `tmp`, was just thinking that or something similar (e.g. we have our `crunchyadm` directory that isn't used much).  I know there have been questions about our various `emptyDirs` in the past, so figured I'd ask (I also tend to think of standalone containers and portability, but that doesn't apply so much with this container).

Definitely functionally the same, so I'm good either way.  Thanks!